corehq/apps/app_manager/models.py

```diff
@@ -2722,14 +2783,6 @@ def get_case_updates(self, case_type):
 
         return updates.union(scheduler_updates)
 
-    @property
-    def case_references(self):
```

Comment:
@millerdev I assume there was no major reason you added these other than just trying to be defensive, yeah? [context](https://github.com/dimagi/commcare-hq/commit/62aaf78675f8d96605016e9acf52ff7d060f7c50)